avocado/utils/multipath.py

```diff
@@ -106,6 +106,19 @@ def get_multipath_wwids():
     return wwids
 
 
+def get_multipath_wwid(mpath):
+    """
+    Get the wwid binding for given mpath name
+
```

Comment:
doc string for input `parm` as well, please

Reply:
Also, try to use a multi-line docstring following the PEP-0257:

 https://www.python.org/dev/peps/pep-0257/

Reply:
Multiline comment added.